extensions/machine-learning-services/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "machineLearningServices",
+  "displayName": "%displayName%",
+  "description": "%description%",
+  "version": "1.0.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+    "vscode": "^1.25.0",
+    "azdata": ">=1.13.0"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+  "icon": "images/ML_ExtensionIcon.png",
+  "aiKey": "AIF-37eefaf0-8022-4671-a3fb-64752724682e",
+  "activationEvents": [
+    "*"
```

Comment:
Avoid * activation events if you can

Reply:
following mssql and notebook extensions. not sure what to put here. the extension is not for specific language 

---
In reply to: [357335173](https://github.com/microsoft/azuredatastudio/pull/8622#discussion_r357335173) [](ancestors = 357335173)

Reply:
If your extension has no reason to be launch on startup, the most straight forward activation event is a command one, so in your case something like `"activationEvents": ["onCommand:ml.command.managePackages"],`

Reply:
it does need to launch at start up. maybe not in this PR but I'm going to add dashboard so. technically the extension is needed when there's a connection 

---
In reply to: [357377118](https://github.com/microsoft/azuredatastudio/pull/8622#discussion_r357377118) [](ancestors = 357377118)

Reply:
What do you mean by add a dashboard? the official dashboard is just json contributions, so that shouldn't need you to be started.

Reply:
I'm just saying you said activate the extension on this command: managePackages
I'm saying I need the extension to active even if that command is not executed or maybe I misunderstood you

---
In reply to: [357435464](https://github.com/microsoft/azuredatastudio/pull/8622#discussion_r357435464) [](ancestors = 357435464)